sdn_tests/testplans/GPINs-Brixia-gNMI-Wildcard-Subscription-Test-Plan.md

```diff
@@ -0,0 +1,10710 @@
+# **Objective**
+
+This document outlines gNMI wildcard subscription feature testing for GPINs on Brixia, and defines test cases for all paths that need wildcards subscription for SB400 switch.
```

Comment:
This document outlines gNMI wildcard subscription feature testing for Sonic-Gnmi, and defines test cases for paths that support wildcards subscription.

Reply:
Also remove GPINs-Brixia from doc title